src/Tracy/Debugger/Debugger.php

```diff
@@ -319,8 +319,12 @@ public static function exceptionHandler(\Throwable $exception): void
 					require self::$errorTemplate ?: __DIR__ . '/assets/error.500.phtml';
 				})(empty($e));
 			} elseif (PHP_SAPI === 'cli') {
-				@fwrite(STDERR, 'ERROR: application encountered an error and can not continue. '
-					. (isset($e) ? "Unable to log error.\n" : "Error was logged.\n")); // @ triggers E_NOTICE when strerr is closed since PHP 7.4
+				// @ triggers E_NOTICE when strerr is closed since PHP 7.4
```

Comment:
there's a typo `strerr` should be `stderr`

Reply:
serepes